views/wrapper.php

```diff
@@ -51,4 +51,6 @@
 /**
  * $this in this context is an instance of View class
  */
-do_action( 'advanced-cron-manager/screen/wrap/after', $this ); ?>
+do_action( 'advanced-cron-manager/screen/wrap/after', $this );
+do_action( 'advanced-cron-manager/screen/wrap/after-modal', $this );
```

Comment:
Why duplicate the same action? I suggest just use the `advanced-cron-manager/screen/wrap/after`, the other one doesn't change anything